pkg/regexp/regexp.go

```diff
@@ -0,0 +1,210 @@
+package regexp
```

Comment:
I’m not sure about using the same name as the standard library package, the two are not interoperable in any sense. OTOH there’s a certain elegance to this.

Reply:
Well, for dynaminc use (i.e. create a regexp in a function) you never want the lazy stuff.

Reply:
That’s a good point.